src/App.js

```diff
@@ -1,16 +1,133 @@
 import React, { Component } from 'react'
-import logo from './logo.svg'
 import './App.css'
+import PlayMenu from './PlayMenu'
+import Naming from './Naming'
 
 class App extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      game: {
+        gameState: 'naming',
+        difficulty: 1
+      },
+      progress: {
+        days: 12,
+        miles: 123
+      },
+      inventory: {
+        waterFilter: 12,
+        sleepingBags: 3,
+        food: 200
+      },
+      people: [{
+        name: 'You',
```

Comment:
This is also an important string that should be defined as a `const` (e.g. `const YOU = 'You'`)